index.html

```diff
@@ -305,9 +305,12 @@ <h2>Sponsors</h2>
                     <div style="display: block">
                         <div class="logo-sponsors">
                              <a href="https://www.wijin.tech/" target="_blank" rel="noopener noreferrer">
-                                <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 1374.97 588.52" height="400px" width="400px">
+                                <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 1374.97 588.52" height="150px" width="320px">
```

Comment:
est-ce qu'on est obligés de laisser le paramètre height, il est pas calculé automatiquement à partir de la largeur ?
```suggestion
                                <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 1374.97 588.52" width="320px">
```

Reply:
si je ne mets height, cela fait un carré